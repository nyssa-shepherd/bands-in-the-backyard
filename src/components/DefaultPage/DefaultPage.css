.default-page {
  background-image: url('../../images/concert-home.jpg');
  background-position: bottom;
  background-size: cover;
  background-repeat: no-repeat;
  height: 100vh;
}

form {
  margin: 0 35%;
  padding-top: 120px;
}

input {
  height: 90px;
}

