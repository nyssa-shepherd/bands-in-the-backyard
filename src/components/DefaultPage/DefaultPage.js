import  React from 'react';
import Search from '../../containers/Search/Search.js';
import './DefaultPage.css';

const DefaultPage = () => { 
  return (
    <div className='default-page'>
      <div>
        <h1 className='title'>Bands 
        <span className='in-the'>in the</span> 
          Backyard
        </h1>
      </div>
      <div>
        <p>Enter your location to see upcoming concets near you.</p>
      </div>
      <div>
        <Search />
      </div>
    </div>
  );
};

export default DefaultPage;