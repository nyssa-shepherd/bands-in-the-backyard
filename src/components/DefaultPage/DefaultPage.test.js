import React from 'react';
import { shallow } from 'enzyme';
import DefaultPage from './DefaultPage.js';

describe('DefaultPage', () => {

  it('snapshot test', () => {
    const wrapper = shallow(<DefaultPage />);
    expect(wrapper).toMatchSnapshot();
  });

});