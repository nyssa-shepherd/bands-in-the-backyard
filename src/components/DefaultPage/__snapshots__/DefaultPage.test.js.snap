// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultPage snapshot test 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DefaultPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div>
          <h1
                    className="title"
          >
                    Bands
                    <span
                              className="in-the"
                    >
                              in the
                    </span>
                    Backyard
          </h1>
</div>,
        <div>
          <p>
                    Enter your location to see upcoming concets near you.
          </p>
</div>,
        <div>
          <withRouter(Connect(Search)) />
</div>,
      ],
      "className": "default-page",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1
            className="title"
>
            Bands
            <span
                        className="in-the"
            >
                        in the
            </span>
            Backyard
</h1>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Bands",
              <span
                className="in-the"
>
                in the
</span>,
              "Backyard",
            ],
            "className": "title",
          },
          "ref": null,
          "rendered": Array [
            "Bands",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "in the",
                "className": "in-the",
              },
              "ref": null,
              "rendered": "in the",
              "type": "span",
            },
            "Backyard",
          ],
          "type": "h1",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p>
            Enter your location to see upcoming concets near you.
</p>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Enter your location to see upcoming concets near you.",
          },
          "ref": null,
          "rendered": "Enter your location to see upcoming concets near you.",
          "type": "p",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <withRouter(Connect(Search)) />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div>
            <h1
                        className="title"
            >
                        Bands
                        <span
                                    className="in-the"
                        >
                                    in the
                        </span>
                        Backyard
            </h1>
</div>,
          <div>
            <p>
                        Enter your location to see upcoming concets near you.
            </p>
</div>,
          <div>
            <withRouter(Connect(Search)) />
</div>,
        ],
        "className": "default-page",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1
              className="title"
>
              Bands
              <span
                            className="in-the"
              >
                            in the
              </span>
              Backyard
</h1>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Bands",
                <span
                  className="in-the"
>
                  in the
</span>,
                "Backyard",
              ],
              "className": "title",
            },
            "ref": null,
            "rendered": Array [
              "Bands",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "in the",
                  "className": "in-the",
                },
                "ref": null,
                "rendered": "in the",
                "type": "span",
              },
              "Backyard",
            ],
            "type": "h1",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <p>
              Enter your location to see upcoming concets near you.
</p>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Enter your location to see upcoming concets near you.",
            },
            "ref": null,
            "rendered": "Enter your location to see upcoming concets near you.",
            "type": "p",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <withRouter(Connect(Search)) />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
